src/test/objectstore/Allocator_bench.cc

```diff
@@ -326,6 +326,38 @@ TEST_P(AllocTest, test_alloc_bench_10_300)
   doOverwriteTest(capacity, prefill, overwrite);
 }
 
+TEST_P(AllocTest, mempoolAccounting)
+{
+  uint64_t bytes = mempool::bluestore_alloc::allocated_bytes();
+  uint64_t items = mempool::bluestore_alloc::allocated_items();
+
+  uint64_t alloc_size = 4 * 1024;
+  uint64_t capacity = 512ll * 1024 * 1024 * 1024;
+  Allocator* alloc = Allocator::create(g_ceph_context, string(GetParam()),
+				       capacity, alloc_size);
+  ASSERT_NE(alloc, nullptr);
```

Comment:
might want to transpose these two parameters. see
```c++
#define EXPECT_NE(expected, actual) \
  EXPECT_PRED_FORMAT2(::testing::internal::CmpHelperNE, expected, actual)
```
its ASSERT counterpart is not very obvious regarding this
```c++
#if !GTEST_DONT_DEFINE_ASSERT_NE
# define ASSERT_NE(val1, val2) GTEST_ASSERT_NE(val1, val2)
#endif
```

but they share the same helper.

Reply:
I think that this is correct, because error message print like this:
`Expected equality of these values:
  mempool::bluestore_alloc::allocated_bytes()
    Which is: 496320
  bytes
    Which is: 0`
Otherwise it will print like this:
`Expected equality of these values:
  bytes
    Which is: 0
  bytes
    Which is: 496320`

Reply:
in my case
```
[ RUN      ] Allocator/AllocTest.mempoolAccounting/2
../src/test/objectstore/Allocator_bench.cc:339: Failure
Expected: (alloc) != ((Allocator*)42), actual: 0x2a vs 0x2a
[  FAILED  ] Allocator/AllocTest.mempoolAccounting/2, where GetParam() = "avl" (0 ms)
```